rawspec_file.c

```diff
@@ -6,14 +6,24 @@
 #include <fcntl.h>
 
 #include "rawspec_file.h"
+#include "fbh5_defs.h"
 
-int open_output_file(const char * dest, const char *stem, int output_idx)
+// Open a single Filterbank file for one of the following:
+//   * nants = 0
+//   * a single antenna of a set
+//   * ICS
+int open_output_file(callback_data_t *cb_data, const char * dest, const char *stem, int output_idx, int antenna_index)
```

Comment:
I feel that it would be neater to have the fbh5 implementation as a second function. Can split em up later mind

Reply:
Also can the final arg be renamed to `ics_flag` in order to match the header declaration. Further I'd say it should be a [0, 1] flag.

Reply:
The antenna_index parameter holds one of the following values:
* -1    :: ics case 
* > 0 :: splitant case and the antenna_index value indicates which antenna to use
* = 0 :: nants=0 (not ATA) so use the cb_data->fbh5_ctx_ant[0]

Reply:
Oh, nice, I didn't fully grasp that when reading the code.. perhaps that's the only issue to resolve.